compiler/optimizer/ValuePropagationTable.hpp

```diff
@@ -820,6 +820,14 @@ TR::Node * constrainLongBitCount(OMR::ValuePropagation *vp, TR::Node *node);
 #define sbitpermuteVPHandler constrainChildren
 #define ibitpermuteVPHandler constrainChildren
 #define lbitpermuteVPHandler constrainChildren
+#define lcompressbitsVPHandler constrainChildren
+#define icompressbitsVPHandler constrainChildren
+#define scompressbitsVPHandler constrainChildren
+#define bcompressbitsVPHandler constrainChildren
+#define lexpandbitsVPHandler constrainChildren
+#define iexpandbitsVPHandler constrainChildren
+#define sexpandbitsVPHandler constrainChildren
+#define bexpandbitsVPHandler constrainChildren
```

Comment:
Is it likely that we might have constraints on the operands that would allow constraints to be computed for the result of the operation?

Reply:
For instance, if we knew that the second operand of `icompressbits` had four bits set, the result would be in the range [0,15].